pkg/helm/repo.go

```diff
@@ -224,3 +224,7 @@ func RepoRemove(ctx context.Context, logger logr.Logger, name string) (err error
 	log.Info(fmt.Sprintf("%q has been removed from your repositories", name))
 	return nil
 }
+
+func IsOCIRepo(url string) bool {
```

Comment:
"helm.sh/helm/v3/pkg/registry".IsOCI function

Reply:
The IsOCIRepo function is deleted and all usage has been changed to registry.IsOCI()!